forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -19,21 +20,70 @@ import com.joinforage.forage.android.network.data.CheckBalanceRepository
 import com.joinforage.forage.android.network.model.ForageApiResponse
 import com.joinforage.forage.android.network.model.ForageError
 import com.joinforage.forage.android.ui.ForagePINEditText
+import com.launchdarkly.sdk.LDContext
+import com.launchdarkly.sdk.android.LDClient
+import com.launchdarkly.sdk.android.LDConfig
+import com.launchdarkly.sdk.android.integrations.TestData
 import java.util.UUID
 
+internal object VaultConstants {
+    const val VGS_VAULT_TYPE = "vgs_vault_type"
+    const val BT_VAULT_TYPE = "bt_vault_type"
+}
+
+internal object LDConstants {
+    const val VAULT_TYPE_FLAG = "vault-primary-traffic-percentage"
```

Comment:
can we refactor `VAULT_TYPE_FLAG` to `VAULT_PRIMARY_TRAFFIC_PERCENTAGE_FLAG` or something alike?

we can consider creating a separate wrapper class for each flag to contain the default value and any mapping / fallback / retrieval logic

Reply:
This may make it easier to test and mock the values of flags in our tests.

Reply:
Flag name changed and contexts/flags have been separated [9f2e3dc](https://github.com/teamforage/forage-android-sdk/pull/31/commits/9f2e3dcfba1559e369e43ded24af68f69a99772d). In terms of a wrapper around the flag logic, I think we can punt that down the line for now. There may come a time where that is a smart move, but I don't think adding any new flags to the mobile SDKs is very close on the roadmap.